core/store/orm/schema.go

```diff
@@ -95,6 +95,7 @@ type ConfigSchema struct {
 	MinRequiredOutgoingConfirmations          uint64          `env:"MIN_OUTGOING_CONFIRMATIONS"`
 	MinimumContractPayment                    assets.Link     `env:"MINIMUM_CONTRACT_PAYMENT" default:"1000000000000000000"`
 	MinimumRequestExpiration                  uint64          `env:"MINIMUM_REQUEST_EXPIRATION" default:"300"`
+	MailboxCapacity                           uint64          `env:"MAILBOX_CAPACITY" default:"50"`
```

Comment:
Nit: Can we call it something more descriptive, like `DIRECT_REQUEST_LOG_BUFFER` or something?

Reply:
i thought it would be more general, but yeah